src/app/frontend/replicasetlist/logsmenu.scss

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import '../variables';
+
+.kd-replicaset-card-logs-menu {
+  font-size: $body-font-size-base;
+}
+
+.kd-menu-logs-item-header {
+  box-sizing: inherit;
+  color: $foreground-2;
+  font-size: $body-font-size-base;
+  white-space: nowrap;
+}
+
+.kd-menu-logs-item {
+  box-sizing: inherit;
+  font-size: $body-font-size-base;
+  white-space: nowrap;
+}
+
+.kd-menu-logs-item-since {
+  max-width: 15 * $baseline-grid;
```

Comment:
I think this value could be a little bit higher.

![image](https://cloud.githubusercontent.com/assets/2823399/12574679/5f878cda-c406-11e5-9e0a-d0315d4863ed.png)

![image](https://cloud.githubusercontent.com/assets/2823399/12574696/7e2be7ee-c406-11e5-9ba0-a04c7396970f.png)

As you can see right margin is inconsistent right now.